frontend/javascripts/oxalis/controller/scene_controller.ts

```diff
@@ -335,18 +396,21 @@ class SceneController {
   }
 
   setIsosurfaceVisibility(id: number, visibility: boolean): void {
-    this.isosurfacesGroupsPerSegmentationId[id].visible = visibility;
+    _.forEach(this.isosurfacesGroupsPerSegmentationId[id], (meshGroup) => {
+      meshGroup.visible = visibility;
```

Comment:
Oh, interesting thought. I had to look into the debugger to figure out why not (because it has been some time since I worked on the code 😅)
But that is likely not the case. The CustomLOD manipulates the visibility of the `Object3D` which contains all groups which then contain the meshes chunk-wise. I hope the "illustration" below makes it clearer than trying to explain it in text :)

```
CustomLOD
  - Level 0:
    - levelId = 0
    -  Object3D  <-- visibility manipulated by CustomLOD.update
      - Group <-- visibility manipulated by setIsosurfaceVisibility
        - Mesh Chunk 1
        - Mesh Chunk 2
        - ...
      - Group <-- visibility manipulated by setIsosurfaceVisibility
        - Mesh Chunk 1
        - Mesh Chunk 2
        - ...
  - Level 1 
    - levelId = 0
    -  Object3D <-- visibility manipulated by CustomLOD.update
      - Group <-- visibility manipulated by setIsosurfaceVisibility
        - Mesh Chunk 1
        - Mesh Chunk 2
        - ...

Reply:
Great, thanks for the clarification!